app/forms/state_file/sales_use_tax_form.rb

```diff
@@ -10,6 +10,7 @@ class SalesUseTaxForm < QuestionsForm
     validates :sales_use_tax,
       presence: true,
       numericality: {
+        allow_blank: true,
         greater_than_or_equal_to: 0
```

Comment:
question: why wouldnt you just add one message in the numericality hash do you don't have to have two strings in the yaml? is it to make it more clear?
````
numericality: {
        allow_blank: true,
        greater_than_or_equal_to: 0,
        message:  .not_a_number }
```